lib/components/OpenProjectWorkPackageBlock.tsx

```diff
@@ -289,33 +285,50 @@ const OpenProjectWorkPackageBlockComponent = ({
   );
 };
 
-export default OpenProjectWorkPackageBlockComponent;
+// export default OpenProjectWorkPackageBlockComponent;
 
-export const openProjectWorkPackageBlockSpec = createReactBlockSpec({
+export const openprojectWorkPackageBlockConfig = createBlockConfig(
+  () => ({
     type: "openProjectWorkPackage",
     propSchema: {
-      wpid: { default: "", type: "string" },
-      subject: { default: "", type: "string" },
-      status: { default: "", type: "string" },
-      assignee: { default: "", type: "string" },
-      type: { default: "", type: "string" },
-      href: { default: "", type: "string" },
+      wpid: { default: "" },
+      subject: { default: "" },
+      href: { default: "" },
     },
     content: "inline",
-  },
+  }) as const
+);
+
+export const openProjectWorkPackageBlockSpec = createReactBlockSpec(
+  openprojectWorkPackageBlockConfig,
+  { render: (props) => <OpenProjectWorkPackageBlockComponent block={props.block} editor={props.editor as any} /> }
+);
+
+export const openProjectWorkPackageVanillaBlockSpec = createBlockSpec(
```

Comment:
I think in the drawing from Friday we called this one "static" (instead of "vanilla"). I think static is transporting a bit more meaning?

Reply:
I like that. Let's adjust!